docs/troubleshooting-guide/filters.md

```diff
@@ -22,6 +23,8 @@ It's always a good idea to start with a quick sanity check:
 
 ## Question filters
 
+If a question filter is giving you no results or the wrong results:
```

Comment:
Not a blocker for this PR, but in a future PR we might want to revise this section, as it doesn't distinguish between the different scenarios presented by the query builder vs. the native query editor.

Reply:
Good point! There is a "Special Cases" section below this that points people to "Troubleshooting SQL variables" section, but putting it inline might be better.

https://www.metabase.com/docs/latest/troubleshooting-guide/filters#special-cases